uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -927,6 +963,10 @@ define([
           }
         } else if (this.isMode('create')) {
           this.request.status_id = this.requestStatuses['waiting_approval'].value;
+
+          if (this.directiveOptions.forStaff) {
```

Comment:
`this.request.status_id = this.directiveOptions.forStaff ? this.requestStatuses['approved'].value : this.requestStatuses['waiting_approval'].value`